### DISCUSSION TITLE
必须依靠党和政府解决烂尾楼问题

### DISCUSSION THREAD

**Discussion by allenaij** [WeNeedHome/SummaryOfLoanSuspension#1013] [Ideas]
现在必须依靠党和政府，只能相信党和政府接盘烂尾楼了。
现在某些敌对势力和别有用心的人利用烂尾楼业主的不满情绪，万一真的把党和政府搞垮了，我们这些业主能得到什么好处？
难道指望未来的新政府接盘烂尾楼吗？

**Comment by zhaipro** [comment]
党是需要领导人民发起斗争，让人民团结起来从而解决矛盾，党是不可能自己解决的。所以我能做点什么呢，谁能告诉我，房住不炒？

**Reply by mathLjtao** [reply]
开发商的问题，凭什么党和政府要帮你解决，自己找开发商去

**Reply by ytx0574** [reply]
> 开发商的问题，凭什么党和政府要帮你解决，自己找开发商去

开发商从买地到建房, 政府从里面捞走了多少, 至少买房的钱70%都在政府口袋. 国家规定预售必须政府和银行成立监管资金, 烂尾盘的监管资金几乎都无了, 不找政府不找党, 你觉得应该找谁?

**Comment by wszxl516** [comment]
看到这个我只想笑，笑死我了.

**Reply by tanuk1997** [reply]
笑的我眼泪分叉，这群奴才天天洗地，还洗的这么没水平

**Comment by shawn-yee** [comment]
鲁迅《聪明人和傻子和奴才》
真实写照

**Comment by winnoryder** [comment]
就是它们给老百姓制造的困难，你让它们给你解决问题？你这逻辑我虽然不懂，但是大受震撼

**Comment by TALK1949** [comment]
难道出事前你不相信？既然相信了为啥还烂尾了？难道预售款不是政府职能部门负责监管？你是怎么做到逻辑自洽的？

**Comment by frshgrass** [comment]
他最好做到 要不然吹什么牛逼 我可以吹牛逼就不用还钱吗

**Comment by redapple2021** [comment]
这人的脑袋一定是让驴给踢过了的

**Comment by xujiyou** [comment]
党和政府摸摸你的毛，让你乖，别闹事，然后这事就跟没有一样

**Comment by cangwuwuwu** [comment]
解铃还需系铃人是吧 我笑死

**Comment by volans19** [comment]
现在必须依靠强奸犯，只能相信强奸犯接盘受害者了。
现在某些反强奸势力帮助受害者骂强奸犯，万一真的把强奸犯搞垮了，我们这些被强奸者能得到什么好处？
难道指望未来的新的强奸犯再强奸我们吗？

**Reply by diaohancai** [reply]
hhh

**Comment by Siryedu** [comment]
你是如何做到颅内监禁的？

**Comment by mathLjtao** [comment]
你这帽子扣得，开发商的问题，凭什么党和政府要帮你解决，自己找开发商去

**Reply by clleo888** [reply]
监管银行, 监管开发商是不是政府的责任?
法律不公又是不是政府的责任?
有扣帽子吗?

**Reply by kongji123** [reply]
> 监管银行, 监管开发商是不是政府的责任? 法律不公又是不是政府的责任? 有扣帽子吗?

他基本问题都没去了解过。就乱发表评论

**Reply by Elijah-Wan** [reply]
对呀，严嵩搞出来的改稻为桑，凭什么要嘉靖陛下负责

**Comment by gongyongzhuanghu** [comment]
你是一条好蛆 还是粉红色的

**Comment by frshgrass** [comment]
合着你还要摇尾乞怜 要姥爷行行好 赔你个三瓜两枣？？？ 
权益是斗争过来的，你是老爷家的十世家奴吗？你和政府的关系就是平等的契约关系。你被人骗了 别没有守诺看住你的钱 你也这样？？？还是政府就是你老爷 你是他的十世家奴。家奴的话患上斯德哥尔摩综合症情有可原。所以你是个斯德哥尔摩综合症的神经病，那你这样骂我们维护老爷就说得通了

**Comment by fnnnnng** [comment]
你妈是怎么生出你这么个废物的

**Reply by Siryedu** [reply]
虽然楼主奴性未改，但是也不至于这样骂别人吧

**Reply by clleo888** [reply]
我比较倾向于责任全在楼主, 他妈没做错

**Reply by fnnnnng** [reply]
> 虽然楼主奴性未改，但是也不至于这样骂别人吧

嗯 我的错，看到如此根深蒂固的奴性 没忍住

**Reply by fnnnnng** [reply]
> 我比较倾向于责任全在楼主, 他妈没做错

嗯 我的错，一时没忍住

**Comment by zhw173** [comment]
不是他们自己搞垮自己的么

**Comment by AlphaXu917** [comment]
自己解决自己？罪己诏？

**Comment by redtime1213** [comment]
解决尼玛 不停贷 谁TM会替你解决问题 傻逼东西 还抱有幻想

**Comment by pujfei** [comment]
你如何把自己从地面上举起来？

**Comment by ytx0574** [comment]
不知道从什么时候开始, 标题内容已经成为了一句反讽...

**Reply by kongji123** [reply]
是，楼上都没理解人家的意思

**Comment by ewkyrtas** [comment]
@allenaij  @winnoryder @zhaipro @ytx0574  @clleo888 @AlphaXu917  @cangwuwuwu  @mathLjtao  @TALK1949 @winnoryder 
https://github.com/WeNeedHome/SummaryOfLoanSuspension/tree/main/images/%E8%B4%B5%E5%B7%9E%E7%9C%81/%E8%B4%B5%E9%98%B3%E5%B8%82/_%E4%B8%AD%E5%A4%A9%C2%B7%E5%90%BE%E4%B9%A1

**Reply by ytx0574** [reply]
艾特我们干嘛, 我们都是你这遭遇. /裂开
